test/qtiCommonRenderer/interactions/pci/common/test.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Common</title>
+        <script type="text/javascript" src="/environment/require.js"></script>
+        <script type="text/javascript">
+            require(['/environment/config.js'], function () {
+                require(['qunitEnv'], function () {
+                    define('qtiCustomInteractionContext', [], function () {
+                        return {
+                            createPciInstance: function (typeIdentifier) {
+                                return {
+                                    typeIdentifier,
+                                    initialize(id, dom, properties, assetManager) {
+                                        this.initializeParameters = { id, dom, properties, assetManager };
+                                    }
+                                };
+                            }
+                        };
+                    });
+                    define('taoQtiItem/qtiCommonRenderer/helpers/container', function () {
+                        return {
+                            get(interaction) {
+                                return {
+                                    children() {
+                                        return {
+                                            get() {
+                                                return 'dom for interaction';
+                                            }
+                                        };
+                                    }
+                                };
+                            }
+                        };
+                    });
```

Comment:
It is a simulation of a dom element. The reason why I used it only here, because it is test specific. It could be a real dom element, but I wanted to have something fix in the last return, to be able to check it in my test.